velox/functions/prestosql/tests/ArithmeticTest.cpp

```diff
@@ -699,5 +699,28 @@ TEST_F(ArithmeticTest, truncate) {
   EXPECT_DOUBLE_EQ(truncate(123456789012345678901.23, -21).value(), 0.0);
 }
 
+TEST_F(ArithmeticTest, normalCDF) {
+  const auto normal_cdf = [&](std::optional<double> mean,
+                              std::optional<double> sd,
+                              std::optional<double> value) {
+    return evaluateOnce<double>("normal_cdf(c0, c1, c2)", mean, sd, value);
+  };
+
+  EXPECT_EQ(0.97500210485177963, normal_cdf(0, 1, 1.96));
```

Comment:
Please also add tests for
std::numeric_limits::infinity(), std::numeric_limits::max(), std::numeric_limits::min(), std::numeric_limits::quiet_NaN() for all parameters.